plugin.program.chrome.launcher/resources/language/English/strings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
+  <string id="30001">Add Website</string>
```

Comment:
I've joined the transifex long time ago to actually get the addon there too but ive yet to find (out of my own lazyness) out the proper way to do the change. No problem to do it now for this release too, better sooner than later.